api/policies/hasCardId.js

```diff
@@ -13,11 +13,11 @@ module.exports = function (req, res, next) {
     if (typeof req.body.cardId === 'number') {
       return next();
     }
-    return res.forbidden({
+    return res.badRequest({
```

Comment:
This was sending a 403 instead of 400 because it's one of the oldest policies and frankly I didn't know the difference back when I was writing this. But it's not an authorization issue it's a matter of missing/incorrect request data so it should be 400 instead.